recipes/pygacode/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "pygacode" %}
+{% set version = "0.50" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "90603c244172f1d4f1a424603c6500361a623d046f7da9d7c221616af62caced" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [win or not x86_64]
+
+requirements:
+  build:
+    - python
+    - {{ compiler('fortran') }}
+    - numpy
+  host:
+    - pip
+    - python
+    - numpy
+  run:
+    - python
+    - numpy
+    - openmp  # [linux]
```

Comment:
See OpenMP instructions at https://conda-forge.org/docs/maintainer/knowledge_base.html#openmp

Reply:
Thank you for the comments, they were quite helpful. Unfortunately, using libgomp as per instructions still results in `undefined symbol: omp_get_thread_num` during runtime.  Supposedly I should include `-fopenmp` when building, but I am not sure how to pass that to the numpy extension builder.  Do you happen to have a pointer for that?

Reply:
CC @isuruf

Reply:
@scopatz I was able to resolve this by patching the `setup.py` file to make sure to link `libgomp`